src/pages/en/guides/cms/buttercms.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ButterCMS & Astro
+description: Add content to your Astro project using ButterCMS
+layout: ~/layouts/CMSLayout.astro
+setup: |
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+stub: false
+---
+
+ButterCMS is a headless CMS and blog engine that allows you to publish structured content to use in your project.
+
+## Integrating with Astro
+
+In this section, we'll use the ButterCMS SDK to bring your ButterCMS data into your Astro project.
+
+To get started, you will need to have the following:
+
+### Prerequisites
+
+1. **An Astro project** - If you don't have an Astro project yet, our [Installation guide](/en/install/auto/) will get you up and running in no time.
+
+2. **A ButterCMS account**. If you don't have an account, you can [sign up](https://buttercms.com/join/) for a free trial.
+
+3. **Your ButterCMS API Token** - You can find your API Token on the [Settings](https://buttercms.com/settings/) page.
+
+### Setup
+
+1. Create a `.env` file in the root of your project and add your API token as an environment variable:
+
+```ini title=".env"
+BUTTER_TOKEN=YOUR_API_TOKEN_HERE
+```
+
+:::tip
+Check out [Using environment variables](/en/guides/environment-variables/) for more information about environment variables and `.env` files in Astro.
```

Comment:
```suggestion
See more about [using environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
```
This is a little awkward, but I'd try to work it into the sentence, not state environment variables twice.

Reply:
I'll update this language in the Contentful guide too!